deployments/ticdc/docker/integration-test.Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN ./download-integration-test-binaries.sh master
 RUN ls ./bin
 
 # Download go into /usr/local dir.
-ENV GOLANG_VERSION 1.19
+ENV GOLANG_VERSION 1.20
 ENV GOLANG_DOWNLOAD_URL https://dl.google.com/go/go$GOLANG_VERSION.linux-amd64.tar.gz
 ENV GOLANG_DOWNLOAD_SHA256 464b6b66591f6cf055bc5df90a9750bf5fbc9d038722bb84a9d56a2bea974be6
```

Comment:
Please update sha.

Reply:
@hi-rustin Could you add a CI to test docker build? I think it's very easy to forget to update hash cause people usually do a  batch search and replace.

Reply:
We do not have docker in docker test in our Jenkin CI. So it's difficult to test it. Maybe we can add a comment here for now.

Reply:
CC: @D3Hunter

Reply:
We should let people notice that we need to change the sha here.

Reply:
don't think there's need to verify sha for test, better remove it